esp/esp/accounting/controllers.py

```diff
@@ -232,11 +232,35 @@ def payments_summary(self):
         payments = Transfer.objects.filter(line_item=payment_li_type)
         return (payments.count(), payments.aggregate(total=Sum('amount_dec'))['total'])
 
+    def classify_transfer(self, transfer):
+        """Give a short human-readable description of a transfer.
+
+        Given a transfer, return a short human-readable phrase describing
+        the type of the transfer."""
+
+        line_item = transfer.line_item
+        if line_item == self.default_payments_lineitemtype():
+            return 'Payment'
+        elif line_item == self.default_finaid_lineitemtype():
+            return 'Financial aid'
+        elif line_item == self.default_siblingdiscount_lineitemtype():
+            return 'Sibling discount'
+        elif transfer.destination == self.default_program_account():
+            req_desc = "required" if line_item.required else "optional"
+            return "Cost ({})".format(req_desc)
```

Comment:
The entire line item is also printed in the table. This classification is supposed to supplement that, and to be short so that the columns align reasonably, because when I saw the the line item "Program admission for HSSP Summer 2016 ($40.00)" I didn't know if that meant whether the user needed to pay that or had paid that.